package.json

```diff
@@ -19,7 +19,7 @@
     "clean": "rimraf dist",
     "prerelease": "yarn build",
     "test": "web-test-runner test/**/*.test.js",
-    "lint": "eslint . --ext .ts --ignore-pattern *.d.ts",
+    "lint": "eslint . --ext .js --ext .ts",
```

Comment:
I _believe_ we can even remove the `--ext` arguments

```json
"lint": "eslint .",
```

Reply:
Oh, you are right! Thanks, fixed it in: https://github.com/marcoroth/turbo_power/commit/be8d354d980aa304adddbaa4aee0899e75b3ea88